### PR TITLE
v1 authn doc strings

### PR DESCRIPTION
**Pull request by r3dcrosse**

### CODE REVIEW

File: packages/pangea-sdk/pangea/services/authn/authn.py [hunk 1]
```diff
@@ -602,18 +1278,40 @@ def __init__(
...
-            # https://dev.pangea.cloud/docs/api/authn#reset-password-during-signin
+            # https://pangea.cloud/docs/api/authn#password-reset
+            # - path: authn::/v1/flow/reset/password
             def password(
                 self,
                 flow_id: str,
-                password: str,
+                password: Optional[str] = None,
                 cancel: Optional[bool] = None,
-                cb_state: Optional[str] = None,
```
Comment (r3dcrosse) [inline packages/pangea-sdk/pangea/services/authn/authn.py:612]: @pangea-andrest it looks like /flow/reset/password got updated and no longer has cb_state or cb_code?

Screenshot 2023-06-30 at 10 57 13 AM

Let me know if this is not the case and I can undo my changes here and in `models.py`

File: packages/pangea-sdk/pangea/services/authn/authn.py [hunk 2]
```diff
@@ -61,16 +61,34 @@ def __init__(
...
         def invalidate(self, session_id: str) -> PangeaResponse[m.SessionInvalidateResult]:
+            """
+            Invalidate Session
+
+            Invalidate a session by session ID.
+
+            OperationId: authn_post_v1_session_invalidate
+
+            Args:
+                session_id (str): An ID for a token
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/authn/authn.py:75]:
```suggestion
                session_id (str): An ID for a session
```

File: packages/pangea-sdk/pangea/services/authn/authn.py [hunk 3]
```diff
@@ -80,15 +98,55 @@ def list(
...
         def logout(self, user_id: str) -> PangeaResponse[m.SessionLogoutResult]:
+            """
+            Log out (service token)
+
+            Invalidate all sessions belonging to a user.
+
+            OperationId: authn_post_v1_session_logout
+
+            Args:
+                user_id (str): The identity of a user or a service.
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/authn/authn.py:140]: Not sure if identity and id are the same. Are they? I think sometime we had both and they were different stuff

File: packages/pangea-sdk/pangea/services/authn/authn.py [hunk 4]
```diff
@@ -138,16 +234,36 @@ def __init__(
...
+                """
+                Invalidate Session | Client
+
+                Invalidate a session by session ID using a client token.
+
+                OperationId: authn_post_v1_client_session_invalidate
+
+                Args:
+                    token (str): A user token value
+                    session_id (str): An ID for a token
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/authn/authn.py:249]: for a session?

File: packages/pangea-sdk/pangea/services/authn/authn.py [hunk 5]
```diff
@@ -271,16 +522,34 @@ def create(
...
+            """
+            Delete User
+
+            Delete a user.
+
+            OperationId: authn_post_v1_user_delete
+
+            Args:
+                email (str, optional): An email address
+                id (str, optional): The identity of a user or a service
```
Comment (pangea-andrest) [inline packages/pangea-sdk/pangea/services/authn/authn.py:537]: identity or id?

**Review by pangea-andrest — APPROVED** [review]
Minor comments. Approved!

**Pull request merged**